test/client/logger.test.js

```diff
@@ -4,16 +4,20 @@ import {
   $mockEndpoint,
   patchXmlHttpRequest,
 } from "@krakenjs/sync-browser-mocks/dist/sync-browser-mocks";
+import { describe, it, beforeEach, expect } from "vitest";
 import { ZalgoPromise } from "@krakenjs/zalgo-promise/src";
 
 import { getLogger, insertMockSDKScript } from "../../src";
 
+// TODO: These tests are explicitly testing the underlying beaver logger itself and otherwise nothing really about
```

Comment:
TODO: I do think we should delete this file entirely or move them to beaver logger, I don't think they're bringing value to the sdk-client.